### PR TITLE
:cow: mongo templates

### PR DESCRIPTION
**Pull request by beseven**
A couple of issues/caveats:
- not sure if traefik and c2s-lb requires a 1 folder too - as based on what I've seen - they don't seem to change from 0
- I don't have a healthcheck for the mongo image; looking at rancher catalogue - there are only recipes for health checks for a cluster

### CODE REVIEW

File: templates/c2s-pharmacy-finder/1/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+version: '2'
+
+services:
+
+  c2s-frontend:
+    image: "nhsuk/connecting-to-services:${c2s_docker_image_tag}"
+    environment:
+      NODE_ENV: production
```
Comment (beseven) [inline templates/c2s-pharmacy-finder/1/docker-compose.yml:8]: Mmm - http://unix.stackexchange.com/questions/32001/what-is-m-and-how-do-i-get-rid-of-it 
This is what happens when I Copy+Paste config from @adamgraves-choices

File: templates/c2s-pharmacy-finder/1/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      io.rancher.container.pull_image: always
+    links:
+    - mongodb-pharmacy:mongo
+
+  mongodb-pharmacy:
+    image: "nhsuk/mongodb-pharmacy::${mongodb_pharmacy_docker_image_tag}"
+    labels:
+      traefik.enable: true
+      traefik.domain: ${traefik_domain}
+      traefik.port: 3001
```
Comment (st3v3nhunt) [inline templates/c2s-pharmacy-finder/1/docker-compose.yml]: Port 27017 for Mongo

File: templates/c2s-pharmacy-finder/1/rancher-compose.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    initializing_timeout: 60000
+    interval: 20000
+    strategy: recreate
+    request_line: GET "/nearby?latitude=53.800755&longitude=-1.549077" "HTTP/1.0"
+    reinitializing_timeout: 60000
+  start_on_create: true
+
+mongodb-pharmacy:
+  scale: 1
+  start_on_create: true
```
Comment (st3v3nhunt) [inline templates/c2s-pharmacy-finder/1/rancher-compose.yml]: This health check should be ok for Mongo:

```
health_check:
  response_timeout: 60000
  healthy_threshold: 2
  port: 271017
  unhealthy_threshold: 3
  initializing_timeout: 60000
  interval: 20000
  strategy: recreate
  request_line: ''
  reinitializing_timeout: 60000
```

File: templates/c2s-pharmacy-finder/1/rancher-compose.yml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      type: "string"
+    - variable: "nearbyservices_docker_image_tag"
+      description: "nearby-services-api Docker image tag"
+      label: "nearby-services-api Docker image tag:"
+      required: true
+      default: "latest"
+      type: "string"
+    - variable: "mongodb_pharmacy_docker_image_tag"
+      description: "mongodb_pharmacy Docker image tag"
+      label: "nearby-services-api Docker image tag:"
```
Comment (st3v3nhunt) [inline templates/c2s-pharmacy-finder/1/rancher-compose.yml]: Label value should be `mongodb_pharmacy Docker image tag:` (assuming the same pattern employed as other questions)

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]

File: templates/c2s-pharmacy-finder/1/docker-compose.yml (hunk 2 above)
Comment (beseven) [inline templates/c2s-pharmacy-finder/1/docker-compose.yml]: 👍

**Review by st3v3nhunt — APPROVED** [review]

**Pull request merged**